code_of_conduct.md

```diff
@@ -0,0 +1,7 @@
+Winter Tech Forum Code of Conduct
+=================================
+
+1. Don't be an asshole.
+
+2. If you have an issue, talk to Bruce and/or a trusted WTF conference participant.
```

Comment:
Also, building out a list of a few participants here who are volunteering to act as a "safe contact" would be good, just to provide some options